### PR TITLE
Pre conference content update

### PR DESCRIPTION
**Pull request by hypha**

### CODE REVIEW

**Comment by hypha** [comment]
Missing: 
1. Our 2022 teams
2. Acknowledgement @artcz

**Comment by hypha** [comment]
I spelled COVID wrong, hopefully only in the commit message.

File: data/pages-content/faq.md [hunk 1]
```diff
@@ -180,15 +199,15 @@ A: Travelling can be expected to be complicated during a pandemic. We will aim t
...
 
 ## **Q. Do I need to wear a mask during the conference?**
 
-A. You are not required to wear a mask but please be mindful of people who want to do so. Risk is perceived differently by people and you should respect their choice.
+A. Although you are not legally required to wear a mask, we strongly recommend that you do, whenever possible.
 
 ## **Q. Do I need to keep social distance during the conference?**
 
-A. No, as of 7 March 2022 social distancing is not required. Please respect the choices of your fellow attendees if they ask you to keep your distance from them.
+A. As of 7 March 2022 social distancing is not required. We will have stickers indicating people's comfort level for interaction: Happy to hug; Happy to handshake; Elbow bumps only; I like my distance. Please respect the choices of your fellow attendees if they ask you to keep your distance from them.
```
Comment (Vaibhavs10) [inline data/pages-content/faq.md:210]: If we can then let's emphasis on stickers.

**Review by Vaibhavs10 — CHANGES_REQUESTED** [review]

**Comment by patrick91** [comment]
@hypha maybe we can add a page under info & support?

File: data/pages-content/covid-info.md [hunk 2]
```diff
@@ -29,29 +29,43 @@ undertaking
 [antigen or PCR testing](https://www2.hse.ie/conditions/covid19/testing/) as
 appropriate.
 
-## The Venue rules
+## The rules and protocols by Law & The CCD
 
 For conferences, the majority of COVID-19 restrictions at the venue have been
 lifted:
 
+- The CCD staff will wear face-masks;
```
[the remark this answers is not in the document]
Comment (hypha) [reply data/pages-content/covid-info.md:37]: No, because we decided not to enforce volunteers and will just strongly encourage.

**Pull request merged**